readthedocs.yml

```diff
@@ -11,7 +11,7 @@ python:
 build:
   os: "ubuntu-22.04"
   tools:
-    python: "3.9"
+    python: "3.11"
```

Comment:
Is this intended to be in this diff?

Reply:
I was playing around with attempts to fix the CI issues. This will be reverted.